playbooks/os/openshift-cluster/vars.yml

```diff
@@ -0,0 +1,33 @@
+---
+openstack_network_prefix: 'openshift-ansible-{{ cluster_id }}'
+openstack_network_cidr: 192.168.{{ 1048576 | random % 256 }}.0/24
+openstack_network_external_net: external
+openstack_network_dns:
+  - 8.8.8.8
+  - 8.8.4.4
+openstack_ssh_keypair: "{{ lookup('env', 'LOGNAME') }}_key"
+openstack_ssh_public_key: "{{ lookup('file','~/.ssh/id_rsa.pub') }}"
+openstack_flavor_ram: 2048
```

Comment:
We should probably separate out the flavor configs so that they can be specified differently for masters and nodes.

Also, it would be nice to specify flavor by other attributes other than ram (at least id or name).